node-backend/app/controllers/CoreValueController.js

```diff
@@ -0,0 +1,151 @@
+const db = require("../models/sequelize");
+const CoreValue = db.core_value;
+const yup = require("yup");
```

Comment:
const yup = require("yup");

const db = require("../models/sequelize");
const CoreValue = db.core_value;
Third party modules and our code modules should be grouped separately